### ISSUE TITLE
nest start --watch my-app in monorepo mode displays help while running

### ISSUE THREAD

**Issue by msheakoski** [nestjs/nest-cli#448]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
```
> nest start --watch my-app

Usage: nest <command> [options]

Options:
  -V, --version                                   output the version number
  -h, --help                                      output usage information

Commands:
  new|n [options] [name]                          Generate Nest application
  build [options] [app]                           Build Nest application
  start [options] [app]                           Build Nest application
  generate|g [options] <schematic> [name] [path]  Generate a Nest element
    Available schematics:
      ┌───────────────┬─────────────┐
      │ name          │ alias       │
      │ application   │ application │
      │ angular-app   │ ng-app      │
      │ class         │ cl          │
      │ configuration │ config      │
      │ controller    │ co          │
      │ decorator     │ d           │
      │ filter        │ f           │
      │ gateway       │ ga          │
      │ guard         │ gu          │
      │ interceptor   │ in          │
      │ interface     │ interface   │
      │ middleware    │ mi          │
      │ module        │ mo          │
      │ pipe          │ pi          │
      │ provider      │ pr          │
      │ resolver      │ r           │
      │ service       │ s           │
      │ library       │ lib         │
      │ sub-app       │ app         │
      └───────────────┴─────────────┘
  info|i                                          Display Nest CLI details
  update|u [options]                              Update Nest dependencies
  add <library> [args...]                         Add a library

 Info  Webpack is building your sources...

Starting type checking service...
Using 1 worker with 2048MB memory limit
Type checking in progress...
Hash: 8567e3f0dd0cb141339d
Version: webpack 4.41.2
Time: 4797ms
Built at: 11/08/2019 6:51:17 PM
Entrypoint main = apps/my-app/main.js
...
```

## Expected behavior
```
> nest start --watch my-app

 Info  Webpack is building your sources...

Starting type checking service...
Using 1 worker with 2048MB memory limit
Type checking in progress...
Hash: 8567e3f0dd0cb141339d
Version: webpack 4.41.2
Time: 4797ms
Built at: 11/08/2019 6:51:17 PM
Entrypoint main = apps/my-app/main.js
...
```

## Minimal reproduction of the problem with instructions
```
nest new my-project --package-manager yarn
cd my-project
nest generate app my-app
nest start --watch my-app
```

## Environment

```
[System Information]
OS Version     : macOS Catalina
NodeJS Version : v12.12.0
YARN Version    : 1.19.1
[Nest Information]
platform-express version : 6.7.2
common version           : 6.7.2
core version             : 6.7.2
```

**Comment by kamilmysliwiec** [comment]
Use `nest start my-app --watch` instead (order is important)

**Comment by msheakoski** [comment]
Thank you for the reply @kamilmysliwiec!

I still think this is confusing to end users because the actual help in the command says to put the options before the app:
```
Commands:
  start [options] [app]                           Build Nest application
```
I also just noticed that it says "Build Nest application" instead of "Start Nest application"

**Comment by msheakoski** [comment]
The latest release that you published, 6.11.5, looks like it fixes all of the above. Thank you once again, @kamilmysliwiec! 😊

**Comment by kamilmysliwiec** [comment]
Awesome :)
